pkg/sqlmigrations/migrations.go

```diff
@@ -233,6 +233,27 @@ var backwardCompatibleMigrations = []migrationDescriptor{
 		name:   "update system.locations with default location data",
 		workFn: updateSystemLocationData,
 	},
+	{
+		// Introduced in v19.2.
+		name:                "change reports fields from timestamp to timestamptz",
+		includedInBootstrap: true,
+		workFn: func(ctx context.Context, r runner) error {
+			if _, err := r.sqlExecutor.ExecWithUser(ctx, "update-reports-meta-generated", nil, /* txn */
+				security.NodeUser,
+				`ALTER TABLE system.reports_meta ALTER generated TYPE TIMESTAMP WITH TIME ZONE`,
```

Comment:
This can never fail, right? And don't these migrations require idempotency?

Reply:
ALTER TYPE with the same type as what there's already is a no-op, so it's idempotent.